.github/workflows/build.yaml

```diff
@@ -20,18 +20,27 @@ jobs:
     steps:
       - name: Checkout code
         uses: actions/checkout@v3
+      #  with:
+      #    submodules: 'true'
+
+      - uses: actions/cache/restore@v3
+        id: cache
         with:
-          submodules: 'true'
-    
-      - name: Exists OpenDDS
-        id: existsOpenDDS
-        continue-on-error: true
-        run: |
-          python $GITHUB_WORKSPACE/third_party/CommonTools/scripts/utilsftp.py -c check -s ${{ secrets.FTP_STORE_SITE }} -u ${{ secrets.FTP_STORE_USER }} -p ${{ secrets.FTP_STORE_PASSWORD }} -r OpenDDS_Build_3.25.zip -l $GITHUB_WORKSPACE/OpenDDS_Build_3.25.zip
+          lookup-only: true
+          enableCrossOsArchive: true
+          path: ./OutputOpenDDS/
+          key: OpenDDS_3.25
+          
+     # - name: Exists OpenDDS
+     #   id: existsOpenDDS
+     #   continue-on-error: true
+     #   run: |
+     #     python $GITHUB_WORKSPACE/third_party/CommonTools/scripts/utilsftp.py -c check -s ${{ secrets.FTP_STORE_SITE }} -u ${{ secrets.FTP_STORE_USER }} -p ${{ secrets.FTP_STORE_PASSWORD }} -r OpenDDS_Build_3.25.zip -l $GITHUB_WORKSPACE/OpenDDS_Build_3.25.zip
```

Comment:
Clean code

Reply:
Leaved in workflow till completed at least a first release cycle